lib/sigstore/verifier.rb

```diff
@@ -76,21 +81,21 @@ def verify(materials:, policy:)
       begin
         # TODO: should this instead be an input to the verify method?
         # See https://docs.google.com/document/d/1kbhK2qyPPk8SLavHzYSDM8-Ueul9_oxIMVFuWMWKz0E/edit?disco=AAABQVV-gT0
-        entry = materials.find_rekor_entry(@rekor_client)
+        entry = find_rekor_entry(bundle, input.hashed_input, offline: offline)
       rescue Sigstore::Error::MissingRekorEntry
         return VerificationFailure.new("Rekor entry not found")
       else
         if entry.inclusion_proof&.checkpoint
           Internal::Merkle.verify_merkle_inclusion(entry)
           Rekor::Checkpoint.verify_checkpoint(@rekor_client, entry)
-        elsif !materials.offline
+        elsif offline
```

Comment:
```suggestion
        elsif !offline
```

the conformance test IMO should pass in verification inputs, which controls things like online/offline verification, and so this should be tested there